00_Core/Scripts/Source/MilkMCM.psc

```diff
@@ -562,12 +562,12 @@ function Page_MilkMaidDebug()
 				float MilkMax = MME_Storage.getMilkMaximum(MaidlistA[MaidIndex])
 				float MilkCnt = MME_Storage.getMilkCurrent(MaidlistA[MaidIndex])
 				float MaidBreastsBaseadjust = MME_Storage.getBreastsBaseadjust(MaidlistA[MaidIndex])
-				float MaidLevel = MME_Storage.getMaidLevel(MaidlistA[MaidIndex])
-				float MilkTick = (MME_Storage.getBreastsBasevalue(MaidlistA[MaidIndex]) + MaidLevel*MaidBoobPerLvl + MaidMilkGen)/3 * (1 + MilkQ.SLA.GetActorArousal(MaidlistA[MaidIndex])/100)
+				int   MaidLevel = MME_Storage.getMaidLevel(MaidlistA[MaidIndex])
+				float MilkTick = (MME_Storage.getBreastsBasevalue(MaidlistA[MaidIndex]) + MaidBoobPerLvl*MaidLevel + MaidMilkGen)/3 * (1 + MilkQ.SLA.GetActorArousal(MaidlistA[MaidIndex])/100)
```

Comment:
Changed the order of operands because multiplying an integer with a float ~might~ result in an integer value. Multiplying a float with an integer is safe.